roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -0,0 +1,85 @@
+---
+- name: Copy Configuration to temporary conf
+  command: >
+    cp {{ openshift.common.config_base }}/master/admin.kubeconfig /tmp/manageiq_admin.kubeconfig
```

Comment:
this should most likely have changed_when: false, to avoid a user thinking that something changed as a result of the run.